koku/masu/processor/parquet/parquet_report_processor.py

```diff
@@ -79,14 +79,12 @@ def __init__(
         ingress_reports_uuid=None,
     ):
         """initialize report processor."""
-        if context is None:
-            context = {}
         self._schema_name = schema_name
         self._provider_uuid = provider_uuid
         self._report_file = Path(report_path)
         self.provider_type = provider_type
         self._manifest_id = manifest_id
-        self._context = context
+        self._context = context or {}
```

Comment:
This is probably fine, but any "falsy" value will now result in `self._context` being initilaized as `{}`.
```suggestion
        self._context = context if context is not None else {}
```

Reply:
I don't really see the difference here. Context should be a dict, not some other falsy value.